BasicMusicPlayer/app/src/main/java/com/example/basicmusicplayer/PlayerActivity.kt

```diff
@@ -92,24 +85,24 @@ class PlayerActivity : AppCompatActivity() {
                         newEntry = true
                     }
 
-                    val updatedFullList = playlistManager.fetchFullPlaylist()
+                    //COMPARISON PART OVER
+
+                    val updatedSublistWithImages = playlistManager.fetchLittlePlaylistWithImages()
                     println("update time")
                     runOnUiThread {
-
-                        // replaces 6 most recent entries with updated order
+                        // replaces 7 most recent entries with updated order
                         if (editPlaylist) {
                             println("only an edit in the playlist")
-                            val rangeToRemove = minOf(6, playlistDetailsList.size)
-                            playlistDetailsList.subList(0, rangeToRemove).clear()
-                            playlistDetailsList.addAll(0, (updatedFullList.subList(0, 6)))
+                            playlistDetailsList.subList(0, 6).clear()
+                            playlistDetailsList.addAll(0, (updatedSublistWithImages.subList(0, 6)))
                             recyclerView.adapter?.notifyItemRangeChanged(0, 6)
                         }
                         // adds new entry to the playlist
                         else if (newEntry) {
                             println("addition to the playlist")
-                            val rangeToRemove = minOf(5, playlistDetailsList.size)
-                            playlistDetailsList.subList(0, rangeToRemove).clear()
-                            playlistDetailsList.addAll(0, (updatedFullList.subList(0, 6)))
+                            //val rangeToRemove = minOf(5, playlistDetailsList.size)
+                            playlistDetailsList.subList(0, 5).clear()
+                            playlistDetailsList.addAll(0, updatedSublistWithImages.subList(0, 6))
```

Comment:
What are these constants for? Could you extract them into variables and name them?